package.json

```diff
@@ -1,8 +1,31 @@
 {
+  "name": "this-week-in-react",
+  "version": "0.0.1",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "start": "react-static start",
+    "stage": "react-static build --staging",
+    "build": "react-static build",
+    "serve": "serve dist -p 3000"
+  },
+  "dependencies": {
+    "chokidar": "2.0.4",
+    "htmr": "0.7.0",
```

Comment:
What is this dependency used for? I don't see it referenced anywhere.

Reply:
It converts html to react, which I was using before but I removed that. I forgot to remove it from the package file. I will remove it in the next PR.